File: app/src/main/java/com/example/csotelo/ashstrolling/core/AshStroll.java
package com.example.csotelo.ashstrolling.core;

public class AshStroll {
    public static void main(String[] args) {

        try {
            String path = "";
            if (args.length > 0) {
                path = args[0];
            }
            Ash ash = new Ash();
            int totalPokemon = ash.catchPokemonInPath(path.toLowerCase());
            System.out.println("Ash cought " + totalPokemon + " pokemon!");

        } catch (Exception e) {
            e.printStackTrace();
        }
    }
}

File: app/src/main/java/com/example/csotelo/ashstrolling/core/AshStroll.kt
package com.example.csotelo.ashstrolling.core

object AshStroll {
    @JvmStatic
    fun main(args: Array<String>) {

        try {
            var path = ""
            if (args.isNotEmpty()) {
                path = args[0]
            }
            val ash = Ash()
            val totalPokemon = ash.catchPokemonInPath(path.toLowerCase())
            println("Ash cought $totalPokemon pokemon!")

        } catch (e: Exception) {
            e.printStackTrace()
        }

    }
}